css/style.css

```diff
@@ -0,0 +1,221 @@
+@import url('https://fonts.googleapis.com/css2?family=Abhaya+Libre:wght@400;500;700&family=Roboto:wght@300;400;500;700&display=swap');
+
+* {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    font-family: 'Roboto', sans-serif;
+    font-size: 14px;
+    font-weight: 400;
+}
+
+a {
+    color: #111111;
+    text-decoration: none;
+}
+
+.container {
+    max-width: 960px;
+    margin: 0px auto;
+}
+
+.postContain {
+    position: relative;
+    padding: 35px 0px 21px 40px;
+    background: #fefefe;
+}
+
+.postContain::after {
+    content: "";
+    width: 1px;
+    height: 100%;
+    position: absolute;
+    left: 0;
+    top: 0;
+    background: #6A6A6A;
```

Comment:
In our coding convention, hex code for color using lower-case characters and numbers. Fix this to #6a6a6a. 
Find and edit similar error in your files.

Reply:
Ok ok. I fixed.